### PR TITLE
Cleaned Up frmScreenRecorder.cs (renamed), frmScriptVariables.cs and frmSettings.cs

### PR DESCRIPTION
**Pull request by openbots-ff**
Cleaned Up frmScreenRecorder.cs (renamed), frmScriptVariables.cs and frmSettings.cs. Making a note to revisit frmScreenRecorder.cs later as it seems to be missing a lot of necessary functionality.

### CODE REVIEW

File: taskt/UI/Forms/frmScreenRecorder.cs [hunk 1]
```diff
@@ -1,93 +1,68 @@
...
+using taskt.Properties;
 
 namespace taskt.UI.Forms
 {
-    public partial class frmSequenceRecorder : UIForm
+    public partial class frmScreenRecorder : UIForm
     {
-
-        public static List<Core.Automation.Commands.ScriptCommand> commandList;
         public frmScriptBuilder callBackForm { get; set; }
```
Comment (ob-ab) [inline taskt/UI/Forms/frmScreenRecorder.cs]: Please use PascalCase for public properties.

File: taskt/UI/Forms/frmScreenRecorder.Designer.cs [hunk 2]
```diff
@@ -60,9 +60,10 @@ private void InitializeComponent()
             this.chkGroupMovesIntoSequences.CheckState = System.Windows.Forms.CheckState.Checked;
             this.chkGroupMovesIntoSequences.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
             this.chkGroupMovesIntoSequences.ForeColor = System.Drawing.Color.AliceBlue;
-            this.chkGroupMovesIntoSequences.Location = new System.Drawing.Point(9, 223);
+            this.chkGroupMovesIntoSequences.Location = new System.Drawing.Point(12, 274);
+            this.chkGroupMovesIntoSequences.Margin = new System.Windows.Forms.Padding(4, 4, 4, 4);
             this.chkGroupMovesIntoSequences.Name = "chkGroupMovesIntoSequences";
-            this.chkGroupMovesIntoSequences.Size = new System.Drawing.Size(281, 19);
+            this.chkGroupMovesIntoSequences.Size = new System.Drawing.Size(348, 24);
```
Comment (ob-ab) [inline taskt/UI/Forms/frmScreenRecorder.Designer.cs:66]: Please make sure that property values of the controls remain unchanged while interacting with Forms.

File: taskt/UI/Forms/frmSettings.cs [hunk 3]
```diff
@@ -152,96 +144,90 @@ private void uiBtnOpen_Click(object sender, EventArgs e)
...
                         MessageBox.Show("taskt-updater.exe not found in resources directory!");
                         return;
                     }
                     else
                     {
-                        System.IO.File.Copy(updaterExecutionResources, updaterExecutableDestination);
+                        File.Copy(updaterExecutionResources, updaterExecutableDestination);
                     }
 
                     var updateProcess = new System.Diagnostics.Process();
```
Comment (ob-ab) [inline taskt/UI/Forms/frmSettings.cs]: Please cleanup `System.Diagnostics` namespace reference

**Review by ob-ab — COMMENTED** [review]
Please make a few small changes and everything else looks good to me.

File: taskt/UI/Forms/frmScreenRecorder.cs (hunk 1 above)
Comment (openbots-ff) [reply taskt/UI/Forms/frmScreenRecorder.cs]: done

File: taskt/UI/Forms/frmSettings.cs (hunk 3 above)
Comment (openbots-ff) [reply taskt/UI/Forms/frmSettings.cs]: done

File: taskt/UI/Forms/frmScreenRecorder.Designer.cs (hunk 2 above)
Comment (openbots-ff) [reply taskt/UI/Forms/frmScreenRecorder.Designer.cs:66]: These values appear to have been auto scaled when I made changes to the label names. I attempted to manually resize them to their original sizes, but all it did was auto move the elements to make room at the following code build.

Comment (ob-ab) [reply taskt/UI/Forms/frmScreenRecorder.Designer.cs:66]: I experienced the same but I somehow managed to change control names without letting it auto scale controls.

**Review by ob-ab — DISMISSED** [review]

**Review by melugi — DISMISSED** [review]
Looks good

File: taskt/UI/Forms/frmSettings.cs [hunk 4]
```diff
@@ -417,16 +387,16 @@ private void btnLaunchDisplayManager_Click(object sender, EventArgs e)
         {
             Supplemental.frmDisplayManager displayManager = new Supplemental.frmDisplayManager();
```
Comment (ob-ab) [inline taskt/UI/Forms/frmSettings.cs]: Please add a using reference to `Supplemental` and cleanup this namespace from code.

**Review by CameronHerwig — DISMISSED** [review]

**Review by ob-ab — COMMENTED** [review]
Please make a small requested change.

File: taskt/UI/Forms/frmSettings.cs (hunk 4 above)
Comment (openbots-ff) [reply taskt/UI/Forms/frmSettings.cs]: done

**Review by ob-ab — APPROVED** [review]
Looks good

**Review by melugi — APPROVED** [review]

**Pull request merged**